moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -751,17 +750,19 @@ bool JogCalcs::addJointIncrements(sensor_msgs::JointState& output, const Eigen::
 
 void JogCalcs::removeDimension(Eigen::MatrixXd& jacobian, Eigen::VectorXd& delta_x, unsigned int row_to_remove)
 {
-  unsigned int num_rows = jacobian.rows() - 1;
-  unsigned int num_cols = jacobian.cols();
+  /*
```

Comment:
Why did you remove this?

Reply:
Oops, it was commented for debugging. Will restore.